README.md

```diff
@@ -98,10 +98,46 @@ A problem occurred while evaluating entry:
 
 #### Properties inheritance
 It might be useful to have properties files that can recursively include
-other properties files (specified via an `include` property).
+other properties files (specified via an `include` property with including file path as value).
 Inherited properties can be overridden by the including set, just redefine
 the property in the file and its value will be used instead of the one
-from the included set.
+from the included set, eg:
```

Comment:
no need for this addition. If you like it, could you change it to:

---
...from the included set.

For example, given a base properties file `base.properties`:

```properties
 foo=bar
 aKey=aValue
 ```

you can override values and add additional ones in another properties file, `debug.properties`:

```properties
 include=/path/to/base.properties
 aKey=overriddenPreviousValue
 aNewKey=aNewValue
 ```

---